src/app/api/items/route.ts

```diff
@@ -0,0 +1,51 @@
+import { connectToDataBase } from '@/lib/mongodb'
+import { IItem, Item } from '@/app/models/Item'
```

Comment:
Why use two "I" to describe the name of a type/interface?

Reply:
one for the Interface followed the name of the inferface